<?xml version="1.0" encoding="utf-8"?>
<TestRun id="94101244-c9fc-4ed4-9fc5-0ef54fe40b72" name="@fastdev 2025-07-25 17:42:25" xmlns="http://microsoft.com/schemas/VisualStudio/TeamTest/2010">
  <Times creation="2025-07-25T17:42:25.5290384+02:00" queuing="2025-07-25T17:42:25.5290385+02:00" start="2025-07-25T17:42:23.5278431+02:00" finish="2025-07-25T17:42:26.3208879+02:00" />
  <TestSettings name="default" id="7e46647e-dd37-452d-a3dc-1658e25bf759">
    <Deployment runDeploymentRoot="_fastdev_2025-07-25_17_42_25" />
  </TestSettings>
  <Results>
    <UnitTestResult executionId="e9d941b1-4978-46b9-83d5-eccd18f4783a" testId="881cf5f8-0024-2c5c-2a49-74df150f2eda" testName="AuthService.Tests.IntegrationTests.AuthServiceIntegrationTests.LoginAsync_ShouldReturnUnauthorized_WhenCredentialsAreInvalid" computerName="fastdev" duration="00:00:00.0082722" startTime="2025-07-25T17:42:25.5890603+02:00" endTime="2025-07-25T17:42:25.5982527+02:00" testType="13cdc9d9-ddb5-4fa4-a97d-d965ccfc6d4b" outcome="Passed" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" relativeResultsDirectory="e9d941b1-4978-46b9-83d5-eccd18f4783a" />
    <UnitTestResult executionId="29dfa938-6771-467a-80e8-f15db1340c44" testId="2086eb0e-57e6-40f6-2533-82d5f19ffe7c" testName="AuthService.Tests.IntegrationTests.AuthServiceIntegrationTests.LoginAsync_ShouldReturnUnauthorized_WhenUserDoesNotExist" computerName="fastdev" duration="00:00:01.4306978" startTime="2025-07-25T17:42:24.1490168+02:00" endTime="2025-07-25T17:42:25.5862696+02:00" testType="13cdc9d9-ddb5-4fa4-a97d-d965ccfc6d4b" outcome="Passed" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" relativeResultsDirectory="29dfa938-6771-467a-80e8-f15db1340c44" />
    <UnitTestResult executionId="ae1c37f3-11c2-4a3d-8093-1e37ef687478" testId="ecad8ccf-6a94-e493-6745-d2b4993e1207" testName="AuthService.Tests.UnitTests.AuthServiceTests.GenerateJwtToken_ShouldReturnValidToken_WhenPrivateKeyIsMocked" computerName="fastdev" duration="00:00:00.2463427" startTime="2025-07-25T17:42:24.1474942+02:00" endTime="2025-07-25T17:42:24.4026049+02:00" testType="13cdc9d9-ddb5-4fa4-a97d-d965ccfc6d4b" outcome="Passed" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" relativeResultsDirectory="ae1c37f3-11c2-4a3d-8093-1e37ef687478" />
    <UnitTestResult executionId="a43816e6-50a5-4a10-9938-c23cc41b83ba" testId="0669a221-5dd7-d122-de95-207b44e6444b" testName="AuthService.Tests.IntegrationTests.AuthServiceTokenValidationTests.ValidateToken_ShouldReturnUnauthorized_WhenTokenIsInvalid" computerName="fastdev" duration="00:00:00.0199928" startTime="2025-07-25T17:42:25.5891626+02:00" endTime="2025-07-25T17:42:25.6092383+02:00" testType="13cdc9d9-ddb5-4fa4-a97d-d965ccfc6d4b" outcome="Passed" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" relativeResultsDirectory="a43816e6-50a5-4a10-9938-c23cc41b83ba" />
    <UnitTestResult executionId="3c6ff4b8-5c92-4ba3-befc-057855364861" testId="ba7f2641-9178-533a-2be7-8870ad325188" testName="AuthService.Tests.UnitTests.AuthServiceTests.LoginAsync_ShouldThrowUnauthorizedAccessException_WhenCredentialsAreInvalid" computerName="fastdev" duration="00:00:00.0283116" startTime="2025-07-25T17:42:24.4357145+02:00" endTime="2025-07-25T17:42:24.4629870+02:00" testType="13cdc9d9-ddb5-4fa4-a97d-d965ccfc6d4b" outcome="Passed" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" relativeResultsDirectory="3c6ff4b8-5c92-4ba3-befc-057855364861" />
    <UnitTestResult executionId="da5e0c6b-e908-47e8-bebc-c31f28a4c0eb" testId="262e833b-2d68-c921-e3d8-2629f30c928c" testName="AuthService.Tests.UnitTests.AuthServiceTests.LoginAsync_ShouldReturnToken_WhenCredentialsAreValid" computerName="fastdev" duration="00:00:00.0314260" startTime="2025-07-25T17:42:24.4288192+02:00" endTime="2025-07-25T17:42:24.4345139+02:00" testType="13cdc9d9-ddb5-4fa4-a97d-d965ccfc6d4b" outcome="Passed" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" relativeResultsDirectory="da5e0c6b-e908-47e8-bebc-c31f28a4c0eb" />
    <UnitTestResult executionId="1970ad9f-6310-4e9b-a18f-0c171d168d7d" testId="d4944854-b93a-7cbc-bc05-bea6dfcefd84" testName="AuthService.Tests.IntegrationTests.AuthServiceRegistrationTests.RegisterAsync_ShouldCreateUser_WhenDataIsValid" computerName="fastdev" duration="00:00:02.0442657" startTime="2025-07-25T17:42:24.1489345+02:00" endTime="2025-07-25T17:42:26.1949337+02:00" testType="13cdc9d9-ddb5-4fa4-a97d-d965ccfc6d4b" outcome="Passed" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" relativeResultsDirectory="1970ad9f-6310-4e9b-a18f-0c171d168d7d" />
    <UnitTestResult executionId="8de38ee2-9fbc-4a1e-b2d7-9f03fdb88955" testId="29e4b15c-a30e-aed2-7c69-a253b23fd5cb" testName="AuthService.Tests.IntegrationTests.AuthServiceTokenValidationTests.ValidateToken_ShouldReturnOk_WhenTokenIsValid" computerName="fastdev" duration="00:00:01.4314999" startTime="2025-07-25T17:42:24.1489843+02:00" endTime="2025-07-25T17:42:25.5857867+02:00" testType="13cdc9d9-ddb5-4fa4-a97d-d965ccfc6d4b" outcome="Passed" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" relativeResultsDirectory="8de38ee2-9fbc-4a1e-b2d7-9f03fdb88955" />
    <UnitTestResult executionId="a7d4bf07-1dfe-4839-a39d-707179139751" testId="8439c253-b70a-13aa-b8ae-3d06c223d26c" testName="AuthService.Tests.IntegrationTests.AuthServiceIntegrationTests.LoginAsync_ShouldReturnToken_WhenCredentialsAreValid" computerName="fastdev" duration="00:00:00.1184663" startTime="2025-07-25T17:42:25.5989114+02:00" endTime="2025-07-25T17:42:25.7196004+02:00" testType="13cdc9d9-ddb5-4fa4-a97d-d965ccfc6d4b" outcome="Failed" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" relativeResultsDirectory="a7d4bf07-1dfe-4839-a39d-707179139751">
      <Output>
        <ErrorInfo>
          <Message>Expected response.IsSuccessStatusCode to be True, but found False.</Message>
          <StackTrace>   at FluentAssertions.Execution.LateBoundTestFramework.Throw(String message)
   at FluentAssertions.Execution.DefaultAssertionStrategy.HandleFailure(String message)
   at FluentAssertions.Execution.AssertionScope.AddPreFormattedFailure(String formattedFailureMessage)
   at FluentAssertions.Execution.AssertionChain.FailWith(Func`1 getFailureReason)
   at FluentAssertions.Execution.AssertionChain.FailWith(Func`1 getFailureReason)
   at FluentAssertions.Execution.AssertionChain.FailWith(String message, Object[] args)
   at FluentAssertions.Primitives.BooleanAssertions`1.BeTrue(String because, Object[] becauseArgs)
   at AuthService.Tests.IntegrationTests.AuthServiceIntegrationTests.LoginAsync_ShouldReturnToken_WhenCredentialsAreValid() in /home/m/development/DatingApp/auth-service/src/AuthService.Tests/Tests/IntegrationTests/AuthServiceIntegrationTests.cs:line 36
--- End of stack trace from previous location ---</StackTrace>
        </ErrorInfo>
      </Output>
    </UnitTestResult>
    <UnitTestResult executionId="130b68e7-d3d3-47d0-9c19-832a181eae7c" testId="f9eb9ca7-65e0-4684-fe46-79864fdd0d7f" testName="AuthService.Tests.IntegrationTests.AuthServiceRegistrationTests.RegisterAsync_ShouldReturnBadRequest_WhenEmailAlreadyExists" computerName="fastdev" duration="00:00:00.0070793" startTime="2025-07-25T17:42:26.1954414+02:00" endTime="2025-07-25T17:42:26.2043872+02:00" testType="13cdc9d9-ddb5-4fa4-a97d-d965ccfc6d4b" outcome="Failed" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" relativeResultsDirectory="130b68e7-d3d3-47d0-9c19-832a181eae7c">
      <Output>
        <ErrorInfo>
          <Message>System.InvalidOperationException : Sequence contains more than one element</Message>
          <StackTrace>   at System.Linq.ThrowHelper.ThrowMoreThanOneElementException()
   at System.Linq.Enumerable.TryGetSingle[TSource](IEnumerable`1 source, Boolean&amp; found)
   at lambda_method822(Closure)
   at Microsoft.EntityFrameworkCore.InMemory.Query.Internal.InMemoryQueryExpression.ResultEnumerable.GetEnumerator()
   at Microsoft.EntityFrameworkCore.InMemory.Query.Internal.InMemoryShapedQueryCompilingExpressionVisitor.QueryingEnumerable`1.Enumerator.MoveNextHelper()
   at Microsoft.EntityFrameworkCore.InMemory.Query.Internal.InMemoryShapedQueryCompilingExpressionVisitor.QueryingEnumerable`1.Enumerator.MoveNextAsync()
   at Microsoft.EntityFrameworkCore.Query.ShapedQueryCompilingExpressionVisitor.SingleOrDefaultAsync[TSource](IAsyncEnumerable`1 asyncEnumerable, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.ShapedQueryCompilingExpressionVisitor.SingleOrDefaultAsync[TSource](IAsyncEnumerable`1 asyncEnumerable, CancellationToken cancellationToken)
   at Microsoft.AspNetCore.Identity.UserManager`1.FindByEmailAsync(String email)
   at AuthService.Services.AuthService.RegisterAsync(RegisterDto dto) in /home/m/development/DatingApp/auth-service/Services/AuthService.cs:line 32
   at AuthService.Controllers.AuthController.Register(RegisterDto dto) in /home/m/development/DatingApp/auth-service/Controllers/AuthController.cs:line 38
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(ActionContext actionContext, IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.&lt;InvokeActionMethodAsync&gt;g__Awaited|12_0(ControllerActionInvoker invoker, ValueTask`1 actionResultValueTask)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.&lt;InvokeNextActionFilterAsync&gt;g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State&amp; next, Scope&amp; scope, Object&amp; state, Boolean&amp; isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.&lt;InvokeFilterPipelineAsync&gt;g__Awaited|20_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.&lt;InvokeAsync&gt;g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.&lt;InvokeAsync&gt;g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIMiddleware.Invoke(HttpContext httpContext)
   at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext, ISwaggerProvider swaggerProvider)
   at Microsoft.AspNetCore.TestHost.HttpContextBuilder.&lt;&gt;c__DisplayClass23_0.&lt;&lt;SendAsync&gt;g__RunRequestAsync|0&gt;d.MoveNext()
--- End of stack trace from previous location ---
   at Microsoft.AspNetCore.TestHost.ClientHandler.SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at Microsoft.AspNetCore.Mvc.Testing.Handlers.CookieContainerHandler.SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at Microsoft.AspNetCore.Mvc.Testing.Handlers.RedirectHandler.SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Net.Http.HttpClient.&lt;SendAsync&gt;g__Core|83_0(HttpRequestMessage request, HttpCompletionOption completionOption, CancellationTokenSource cts, Boolean disposeCts, CancellationTokenSource pendingRequestsCts, CancellationToken originalCancellationToken)
   at AuthService.Tests.IntegrationTests.AuthServiceRegistrationTests.RegisterAsync_ShouldReturnBadRequest_WhenEmailAlreadyExists() in /home/m/development/DatingApp/auth-service/src/AuthService.Tests/Tests/IntegrationTests/AuthServiceRegistrationTests.cs:line 57
--- End of stack trace from previous location ---</StackTrace>
        </ErrorInfo>
      </Output>
    </UnitTestResult>
  </Results>
  <TestDefinitions>
    <UnitTest name="AuthService.Tests.UnitTests.AuthServiceTests.GenerateJwtToken_ShouldReturnValidToken_WhenPrivateKeyIsMocked" storage="/home/m/development/datingapp/auth-service/src/authservice.tests/bin/debug/net8.0/authservice.tests.dll" id="ecad8ccf-6a94-e493-6745-d2b4993e1207">
      <Execution id="ae1c37f3-11c2-4a3d-8093-1e37ef687478" />
      <TestMethod codeBase="/home/m/development/DatingApp/auth-service/src/AuthService.Tests/bin/Debug/net8.0/AuthService.Tests.dll" adapterTypeName="executor://xunit/VsTestRunner3/netcore/" className="AuthService.Tests.UnitTests.AuthServiceTests" name="GenerateJwtToken_ShouldReturnValidToken_WhenPrivateKeyIsMocked" />
    </UnitTest>
    <UnitTest name="AuthService.Tests.IntegrationTests.AuthServiceRegistrationTests.RegisterAsync_ShouldReturnBadRequest_WhenEmailAlreadyExists" storage="/home/m/development/datingapp/auth-service/src/authservice.tests/bin/debug/net8.0/authservice.tests.dll" id="f9eb9ca7-65e0-4684-fe46-79864fdd0d7f">
      <Execution id="130b68e7-d3d3-47d0-9c19-832a181eae7c" />
      <TestMethod codeBase="/home/m/development/DatingApp/auth-service/src/AuthService.Tests/bin/Debug/net8.0/AuthService.Tests.dll" adapterTypeName="executor://xunit/VsTestRunner3/netcore/" className="AuthService.Tests.IntegrationTests.AuthServiceRegistrationTests" name="RegisterAsync_ShouldReturnBadRequest_WhenEmailAlreadyExists" />
    </UnitTest>
    <UnitTest name="AuthService.Tests.IntegrationTests.AuthServiceIntegrationTests.LoginAsync_ShouldReturnToken_WhenCredentialsAreValid" storage="/home/m/development/datingapp/auth-service/src/authservice.tests/bin/debug/net8.0/authservice.tests.dll" id="8439c253-b70a-13aa-b8ae-3d06c223d26c">
      <Execution id="a7d4bf07-1dfe-4839-a39d-707179139751" />
      <TestMethod codeBase="/home/m/development/DatingApp/auth-service/src/AuthService.Tests/bin/Debug/net8.0/AuthService.Tests.dll" adapterTypeName="executor://xunit/VsTestRunner3/netcore/" className="AuthService.Tests.IntegrationTests.AuthServiceIntegrationTests" name="LoginAsync_ShouldReturnToken_WhenCredentialsAreValid" />
    </UnitTest>
    <UnitTest name="AuthService.Tests.UnitTests.AuthServiceTests.LoginAsync_ShouldReturnToken_WhenCredentialsAreValid" storage="/home/m/development/datingapp/auth-service/src/authservice.tests/bin/debug/net8.0/authservice.tests.dll" id="262e833b-2d68-c921-e3d8-2629f30c928c">
      <Execution id="da5e0c6b-e908-47e8-bebc-c31f28a4c0eb" />
      <TestMethod codeBase="/home/m/development/DatingApp/auth-service/src/AuthService.Tests/bin/Debug/net8.0/AuthService.Tests.dll" adapterTypeName="executor://xunit/VsTestRunner3/netcore/" className="AuthService.Tests.UnitTests.AuthServiceTests" name="LoginAsync_ShouldReturnToken_WhenCredentialsAreValid" />
    </UnitTest>
    <UnitTest name="AuthService.Tests.IntegrationTests.AuthServiceTokenValidationTests.ValidateToken_ShouldReturnUnauthorized_WhenTokenIsInvalid" storage="/home/m/development/datingapp/auth-service/src/authservice.tests/bin/debug/net8.0/authservice.tests.dll" id="0669a221-5dd7-d122-de95-207b44e6444b">
      <Execution id="a43816e6-50a5-4a10-9938-c23cc41b83ba" />
      <TestMethod codeBase="/home/m/development/DatingApp/auth-service/src/AuthService.Tests/bin/Debug/net8.0/AuthService.Tests.dll" adapterTypeName="executor://xunit/VsTestRunner3/netcore/" className="AuthService.Tests.IntegrationTests.AuthServiceTokenValidationTests" name="ValidateToken_ShouldReturnUnauthorized_WhenTokenIsInvalid" />
    </UnitTest>
    <UnitTest name="AuthService.Tests.IntegrationTests.AuthServiceIntegrationTests.LoginAsync_ShouldReturnUnauthorized_WhenCredentialsAreInvalid" storage="/home/m/development/datingapp/auth-service/src/authservice.tests/bin/debug/net8.0/authservice.tests.dll" id="881cf5f8-0024-2c5c-2a49-74df150f2eda">
      <Execution id="e9d941b1-4978-46b9-83d5-eccd18f4783a" />
      <TestMethod codeBase="/home/m/development/DatingApp/auth-service/src/AuthService.Tests/bin/Debug/net8.0/AuthService.Tests.dll" adapterTypeName="executor://xunit/VsTestRunner3/netcore/" className="AuthService.Tests.IntegrationTests.AuthServiceIntegrationTests" name="LoginAsync_ShouldReturnUnauthorized_WhenCredentialsAreInvalid" />
    </UnitTest>
    <UnitTest name="AuthService.Tests.IntegrationTests.AuthServiceIntegrationTests.LoginAsync_ShouldReturnUnauthorized_WhenUserDoesNotExist" storage="/home/m/development/datingapp/auth-service/src/authservice.tests/bin/debug/net8.0/authservice.tests.dll" id="2086eb0e-57e6-40f6-2533-82d5f19ffe7c">
      <Execution id="29dfa938-6771-467a-80e8-f15db1340c44" />
      <TestMethod codeBase="/home/m/development/DatingApp/auth-service/src/AuthService.Tests/bin/Debug/net8.0/AuthService.Tests.dll" adapterTypeName="executor://xunit/VsTestRunner3/netcore/" className="AuthService.Tests.IntegrationTests.AuthServiceIntegrationTests" name="LoginAsync_ShouldReturnUnauthorized_WhenUserDoesNotExist" />
    </UnitTest>
    <UnitTest name="AuthService.Tests.UnitTests.AuthServiceTests.LoginAsync_ShouldThrowUnauthorizedAccessException_WhenCredentialsAreInvalid" storage="/home/m/development/datingapp/auth-service/src/authservice.tests/bin/debug/net8.0/authservice.tests.dll" id="ba7f2641-9178-533a-2be7-8870ad325188">
      <Execution id="3c6ff4b8-5c92-4ba3-befc-057855364861" />
      <TestMethod codeBase="/home/m/development/DatingApp/auth-service/src/AuthService.Tests/bin/Debug/net8.0/AuthService.Tests.dll" adapterTypeName="executor://xunit/VsTestRunner3/netcore/" className="AuthService.Tests.UnitTests.AuthServiceTests" name="LoginAsync_ShouldThrowUnauthorizedAccessException_WhenCredentialsAreInvalid" />
    </UnitTest>
    <UnitTest name="AuthService.Tests.IntegrationTests.AuthServiceTokenValidationTests.ValidateToken_ShouldReturnOk_WhenTokenIsValid" storage="/home/m/development/datingapp/auth-service/src/authservice.tests/bin/debug/net8.0/authservice.tests.dll" id="29e4b15c-a30e-aed2-7c69-a253b23fd5cb">
      <Execution id="8de38ee2-9fbc-4a1e-b2d7-9f03fdb88955" />
      <TestMethod codeBase="/home/m/development/DatingApp/auth-service/src/AuthService.Tests/bin/Debug/net8.0/AuthService.Tests.dll" adapterTypeName="executor://xunit/VsTestRunner3/netcore/" className="AuthService.Tests.IntegrationTests.AuthServiceTokenValidationTests" name="ValidateToken_ShouldReturnOk_WhenTokenIsValid" />
    </UnitTest>
    <UnitTest name="AuthService.Tests.IntegrationTests.AuthServiceRegistrationTests.RegisterAsync_ShouldCreateUser_WhenDataIsValid" storage="/home/m/development/datingapp/auth-service/src/authservice.tests/bin/debug/net8.0/authservice.tests.dll" id="d4944854-b93a-7cbc-bc05-bea6dfcefd84">
      <Execution id="1970ad9f-6310-4e9b-a18f-0c171d168d7d" />
      <TestMethod codeBase="/home/m/development/DatingApp/auth-service/src/AuthService.Tests/bin/Debug/net8.0/AuthService.Tests.dll" adapterTypeName="executor://xunit/VsTestRunner3/netcore/" className="AuthService.Tests.IntegrationTests.AuthServiceRegistrationTests" name="RegisterAsync_ShouldCreateUser_WhenDataIsValid" />
    </UnitTest>
  </TestDefinitions>
  <TestEntries>
    <TestEntry testId="881cf5f8-0024-2c5c-2a49-74df150f2eda" executionId="e9d941b1-4978-46b9-83d5-eccd18f4783a" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
    <TestEntry testId="2086eb0e-57e6-40f6-2533-82d5f19ffe7c" executionId="29dfa938-6771-467a-80e8-f15db1340c44" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
    <TestEntry testId="ecad8ccf-6a94-e493-6745-d2b4993e1207" executionId="ae1c37f3-11c2-4a3d-8093-1e37ef687478" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
    <TestEntry testId="0669a221-5dd7-d122-de95-207b44e6444b" executionId="a43816e6-50a5-4a10-9938-c23cc41b83ba" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
    <TestEntry testId="ba7f2641-9178-533a-2be7-8870ad325188" executionId="3c6ff4b8-5c92-4ba3-befc-057855364861" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
    <TestEntry testId="262e833b-2d68-c921-e3d8-2629f30c928c" executionId="da5e0c6b-e908-47e8-bebc-c31f28a4c0eb" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
    <TestEntry testId="d4944854-b93a-7cbc-bc05-bea6dfcefd84" executionId="1970ad9f-6310-4e9b-a18f-0c171d168d7d" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
    <TestEntry testId="29e4b15c-a30e-aed2-7c69-a253b23fd5cb" executionId="8de38ee2-9fbc-4a1e-b2d7-9f03fdb88955" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
    <TestEntry testId="8439c253-b70a-13aa-b8ae-3d06c223d26c" executionId="a7d4bf07-1dfe-4839-a39d-707179139751" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
    <TestEntry testId="f9eb9ca7-65e0-4684-fe46-79864fdd0d7f" executionId="130b68e7-d3d3-47d0-9c19-832a181eae7c" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
  </TestEntries>
  <TestLists>
    <TestList name="Results Not in a List" id="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
    <TestList name="All Loaded Results" id="19431567-8539-422a-85d7-44ee4e166bda" />
  </TestLists>
  <ResultSummary outcome="Failed">
    <Counters total="10" executed="10" passed="8" failed="2" error="0" timeout="0" aborted="0" inconclusive="0" passedButRunAborted="0" notRunnable="0" notExecuted="0" disconnected="0" warning="0" completed="0" inProgress="0" pending="0" />
    <Output>
      <StdOut>[xUnit.net 00:00:00.00] xUnit.net VSTest Adapter v3.1.0+00ab0a9ec4 (64-bit .NET 8.0.17)
[xUnit.net 00:00:00.09]   Discovering: AuthService.Tests
[xUnit.net 00:00:00.13]   Discovered:  AuthService.Tests
[xUnit.net 00:00:00.16]   Starting:    AuthService.Tests
[xUnit.net 00:00:01.76]       Expected response.IsSuccessStatusCode to be True, but found False.
[xUnit.net 00:00:01.76]       Stack Trace:
[xUnit.net 00:00:01.76]            at FluentAssertions.Execution.LateBoundTestFramework.Throw(String message)
[xUnit.net 00:00:01.76]            at FluentAssertions.Execution.DefaultAssertionStrategy.HandleFailure(String message)
[xUnit.net 00:00:01.76]            at FluentAssertions.Execution.AssertionScope.AddPreFormattedFailure(String formattedFailureMessage)
[xUnit.net 00:00:01.76]            at FluentAssertions.Execution.AssertionChain.FailWith(Func`1 getFailureReason)
[xUnit.net 00:00:01.76]            at FluentAssertions.Execution.AssertionChain.FailWith(Func`1 getFailureReason)
[xUnit.net 00:00:01.76]            at FluentAssertions.Execution.AssertionChain.FailWith(String message, Object[] args)
[xUnit.net 00:00:01.76]            at FluentAssertions.Primitives.BooleanAssertions`1.BeTrue(String because, Object[] becauseArgs)
[xUnit.net 00:00:01.76]         /home/m/development/DatingApp/auth-service/src/AuthService.Tests/Tests/IntegrationTests/AuthServiceIntegrationTests.cs(36,0): at AuthService.Tests.IntegrationTests.AuthServiceIntegrationTests.LoginAsync_ShouldReturnToken_WhenCredentialsAreValid()
[xUnit.net 00:00:01.76]         --- End of stack trace from previous location ---
[xUnit.net 00:00:02.24]       System.InvalidOperationException : Sequence contains more than one element
[xUnit.net 00:00:02.24]       Stack Trace:
[xUnit.net 00:00:02.24]            at System.Linq.ThrowHelper.ThrowMoreThanOneElementException()
[xUnit.net 00:00:02.24]            at System.Linq.Enumerable.TryGetSingle[TSource](IEnumerable`1 source, Boolean&amp; found)
[xUnit.net 00:00:02.24]            at lambda_method822(Closure)
[xUnit.net 00:00:02.24]            at Microsoft.EntityFrameworkCore.InMemory.Query.Internal.InMemoryQueryExpression.ResultEnumerable.GetEnumerator()
[xUnit.net 00:00:02.24]            at Microsoft.EntityFrameworkCore.InMemory.Query.Internal.InMemoryShapedQueryCompilingExpressionVisitor.QueryingEnumerable`1.Enumerator.MoveNextHelper()
[xUnit.net 00:00:02.24]            at Microsoft.EntityFrameworkCore.InMemory.Query.Internal.InMemoryShapedQueryCompilingExpressionVisitor.QueryingEnumerable`1.Enumerator.MoveNextAsync()
[xUnit.net 00:00:02.24]            at Microsoft.EntityFrameworkCore.Query.ShapedQueryCompilingExpressionVisitor.SingleOrDefaultAsync[TSource](IAsyncEnumerable`1 asyncEnumerable, CancellationToken cancellationToken)
[xUnit.net 00:00:02.24]            at Microsoft.EntityFrameworkCore.Query.ShapedQueryCompilingExpressionVisitor.SingleOrDefaultAsync[TSource](IAsyncEnumerable`1 asyncEnumerable, CancellationToken cancellationToken)
[xUnit.net 00:00:02.25]            at Microsoft.AspNetCore.Identity.UserManager`1.FindByEmailAsync(String email)
[xUnit.net 00:00:02.25]         /home/m/development/DatingApp/auth-service/Services/AuthService.cs(32,0): at AuthService.Services.AuthService.RegisterAsync(RegisterDto dto)
[xUnit.net 00:00:02.25]         /home/m/development/DatingApp/auth-service/Controllers/AuthController.cs(38,0): at AuthService.Controllers.AuthController.Register(RegisterDto dto)
[xUnit.net 00:00:02.25]            at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(ActionContext actionContext, IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
[xUnit.net 00:00:02.25]            at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.&lt;InvokeActionMethodAsync&gt;g__Awaited|12_0(ControllerActionInvoker invoker, ValueTask`1 actionResultValueTask)
[xUnit.net 00:00:02.25]            at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.&lt;InvokeNextActionFilterAsync&gt;g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
[xUnit.net 00:00:02.25]            at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
[xUnit.net 00:00:02.25]            at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State&amp; next, Scope&amp; scope, Object&amp; state, Boolean&amp; isCompleted)
[xUnit.net 00:00:02.25]            at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
[xUnit.net 00:00:02.25]         --- End of stack trace from previous location ---
[xUnit.net 00:00:02.25]            at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.&lt;InvokeFilterPipelineAsync&gt;g__Awaited|20_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
[xUnit.net 00:00:02.25]            at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.&lt;InvokeAsync&gt;g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
[xUnit.net 00:00:02.25]            at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.&lt;InvokeAsync&gt;g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
[xUnit.net 00:00:02.25]            at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
[xUnit.net 00:00:02.25]            at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
[xUnit.net 00:00:02.25]            at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIMiddleware.Invoke(HttpContext httpContext)
[xUnit.net 00:00:02.25]            at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext, ISwaggerProvider swaggerProvider)
[xUnit.net 00:00:02.25]            at Microsoft.AspNetCore.TestHost.HttpContextBuilder.&lt;&gt;c__DisplayClass23_0.&lt;&lt;SendAsync&gt;g__RunRequestAsync|0&gt;d.MoveNext()
[xUnit.net 00:00:02.25]         --- End of stack trace from previous location ---
[xUnit.net 00:00:02.25]            at Microsoft.AspNetCore.TestHost.ClientHandler.SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)
[xUnit.net 00:00:02.25]            at Microsoft.AspNetCore.Mvc.Testing.Handlers.CookieContainerHandler.SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)
[xUnit.net 00:00:02.25]            at Microsoft.AspNetCore.Mvc.Testing.Handlers.RedirectHandler.SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)
[xUnit.net 00:00:02.25]            at System.Net.Http.HttpClient.&lt;SendAsync&gt;g__Core|83_0(HttpRequestMessage request, HttpCompletionOption completionOption, CancellationTokenSource cts, Boolean disposeCts, CancellationTokenSource pendingRequestsCts, CancellationToken originalCancellationToken)
[xUnit.net 00:00:02.25]         /home/m/development/DatingApp/auth-service/src/AuthService.Tests/Tests/IntegrationTests/AuthServiceRegistrationTests.cs(57,0): at AuthService.Tests.IntegrationTests.AuthServiceRegistrationTests.RegisterAsync_ShouldReturnBadRequest_WhenEmailAlreadyExists()
[xUnit.net 00:00:02.25]         --- End of stack trace from previous location ---
[xUnit.net 00:00:02.25]   Finished:    AuthService.Tests
</StdOut>
    </Output>
    <RunInfos>
      <RunInfo computerName="fastdev" outcome="Error" timestamp="2025-07-25T17:42:25.7225413+02:00">
        <Text>[xUnit.net 00:00:01.76]     AuthService.Tests.IntegrationTests.AuthServiceIntegrationTests.LoginAsync_ShouldReturnToken_WhenCredentialsAreValid [FAIL]</Text>
      </RunInfo>
      <RunInfo computerName="fastdev" outcome="Error" timestamp="2025-07-25T17:42:26.2055541+02:00">
        <Text>[xUnit.net 00:00:02.24]     AuthService.Tests.IntegrationTests.AuthServiceRegistrationTests.RegisterAsync_ShouldReturnBadRequest_WhenEmailAlreadyExists [FAIL]</Text>
      </RunInfo>
    </RunInfos>
  </ResultSummary>
</TestRun>